src/ext/ComPlus/ca/cpapprolesched.cpp

```diff
@@ -6,17 +6,17 @@
 // sql queries
 
 LPCWSTR vcsApplicationRoleQuery =
-    L"SELECT `ApplicationRole`, `Application_`, `Component_`, `Name` FROM `ComPlusApplicationRole`";
+    L"SELECT `ApplicationRole`, `Application_`, `Component_`, `Name` FROM `Wix4ComPlusApplicationRole`";
 enum eApplicationRoleQuery { arqApplicationRole = 1, arqApplication, arqComponent, arqName };
 
 LPCWSTR vcsUserInApplicationRoleQuery =
-    L"SELECT `UserInApplicationRole`, `ApplicationRole_`, `ComPlusUserInApplicationRole`.`Component_`, `Domain`, `Name` FROM `ComPlusUserInApplicationRole`, `User` WHERE `User_` = `User`";
+    L"SELECT `UserInApplicationRole`, `ApplicationRole_`, `ComPlusUserInApplicationRole`.`Component_`, `Domain`, `Name` FROM `Wix4ComPlusUserInApplicationRole`, `Wix4User` WHERE `User_` = `User`";
```

Comment:
If this should be `Wix4User`, then it appears the following `User` and `Group` tables also need to be `Wix4`:

https://github.com/wixtoolset/wix4/blob/a4314c9a0e446cb7eeff4c9066818503942913d7/src/ext/ComPlus/ca/cpapprolesched.cpp#L15
https://github.com/wixtoolset/wix4/blob/a4314c9a0e446cb7eeff4c9066818503942913d7/src/ext/ComPlus/ca/cppartrolesched.cpp#L15
https://github.com/wixtoolset/wix4/blob/a4314c9a0e446cb7eeff4c9066818503942913d7/src/ext/Msmq/ca/mqqueuesched.cpp#L13
https://github.com/wixtoolset/wix4/blob/a4314c9a0e446cb7eeff4c9066818503942913d7/src/ext/Msmq/ca/mqqueuesched.cpp#L15

Reply:
I thought I caught all those...:sigh: Thanks.